### ISSUE TITLE
lightbox-auto not created when using srcset

### ISSUE THREAD

**Issue by DanielMuller** [ampproject/amphtml#28304]
## What's the issue?
When using srcset in an amp-img tag, lightbox-auto isn't set-up.
All conditions for lightbox-auto to work have been met.

## What isn't working?
When using images with srcset, lightbox isn't autmatically created with lighbox-auto.

## What is the expected behavior?
Images defined with srcset should have a lightbox created (with lightbox-auto), same as images with only src

## How do we reproduce the issue?

Create an amp-img with srcset and one without, as done here: https://jsbin.com/mutecek/edit?html,output

## What browsers are affected?
Tested in 2 browsers. Both failing. I guess this will fail on all browsers.

Google Chrome: 81.0.4044.138 (Official Build) (64-bit)
Revision: 8c6c7ba89cc9453625af54f11fd83179e23450fa-refs/branch-heads/4044@{#999}
OS: Linux
JavaScript: V8 8.1.307.32

Firefox: 76.0 (64-bit)

## Which AMP version is affected?
AMP version: 2004252135000

**Comment by caroqliu** [comment]
Hi @DanielMuller thanks for filing this issue. Can you please describe what is the "broken" behavior and what is the expected behavior?

**Comment by DanielMuller** [comment]
I amended the issue to include in a clearer way what is "broken" and what is expected

**Comment by caroqliu** [comment]
It looks like for some viewport sizes the lightbox is triggering in the srcset example, but not for others. @alanorozco to take a look as he is more familiar with the sizing criteria for auto-lightbox behavior.

**Comment by DanielMuller** [comment]
To add more infos. I tested on a desktop browser. But any window size(up to 1920x1080) didn't work.
On a mobile chrome browser, no issue (any orientation).

**Comment by alanorozco** [comment]
This is interesting!

The browser is selecting a lower resolution image on desktop from the `srcset`. Auto-lightbox uses the `.naturalWidth/Height` properties to determine if the image is shrunk in its render box from a higher resolution, and can therefore be expanded without loss.

In this case, since the lightbox already knows how to use a higher-resolution src from `srcset`, auto-lightbox should consider the highest possible intrinsic width in the `srcset` to determine if it's shrunk.
